pyproject.toml

```diff
@@ -26,6 +25,7 @@ markdown = "!=3.3.5"  # Added due an issue with Markdown 3.3.5
 nautobot = { version = "^1.4.0", optional = true }  # Used for local development
 nautobot-capacity-metrics = {version = "*", optional = true }
 python = "^3.8"
+pydantic = "^2.0.0"
```

Comment:
LTM requires pydantic 2.x at this point?

Reply:
if we want to keep using the latest updates from circuit-maintenance-parser, yes. All the users of Nautobot 1.6, without this, won't be able to take the new parsers without this bump